sormas-app/app/src/main/java/de/symeda/sormas/app/component/dialog/LocationDialog.java

```diff
@@ -288,6 +296,16 @@ public void setFacilityFieldsVisible(boolean visible, boolean clearOnHidden) {
 		}
 	}
 
+	public void setContinentFieldsVisible( boolean clearOnHidden) {
```

Comment:
Code formatting

Also, the method name together with the parameter doesn't make clear what the method does. A method called setXXXVisible with a boolean parameter is usually expected to set XXX visible if the boolean parameter is true and invisible when it's false. I'd suggest to rename the method to `updateContinentFieldsVisibility` and remove the parameter because as far as I can see, it's never called with `false` anyway

Another question would be whether it's necessary to call this method from outside; can't it be automatically executed directly in this class?